getting-started/faqs.md

```diff
@@ -4,220 +4,197 @@
+A centralized, fully managed file transfer service for the whole of government (WOG).
 
 CFT is a part of the SG Tech Stack (SGTS) and is hosted on the Government Commercial Cloud (GCC).
 
-**Key Features:**
+Core capabilities:
 
-**Compliant file transfers**
+- Compliant cross-zone file transfers according to IM8
+- Malware scanning with Content Disarm and Reconstruct (CDR)
+- File Encryption for data integrity and security
+- Application Programming Interface (API) access for web and mobile applications
+- Event-driven with file download notifications
 
-CFT complies with Instruction Manual (IM8) Application Infrastructure Architecture Standard (AIAS) policy guidelines.
+**What are the benefits of using CFT?**
 
-**Malware scanning and Content Disarm and Reconstruct (CDR)** 
+CFT can benefit agencies who require to:
 
-Files are scanned for malware during transfer and malicious content is removed by the CDR functionality. Latest Antivirus (AV) updates are automatically applied to your files when they are being scanned.
+- Transfer large files across zones
+- Move files securely
+- Automatically scale based on their transfer needs
+- Customize workflows
+- Audit and monitor their file transfer activity
+- Accelerate digitization of government services
 
-**Large file transfers**
+**Who are the end users?**
 
-Up to 1GB per transaction.
+CFT is available for government agencies, their business partners, and vendors.
 
-**Application Programming Interface (API) access for web and mobile applications** 
+**What are the different integration options?**
 
-Your applications can communicate with CFT through its APIs enabling you to access CFT to send, scan and receive files.
+CFT will be available for the following integration options:
 
-**Event-driven** 
+1. CFT for Backend Systems - For file transfers between systems.
+2. CFT for Web and Mobile Applications - For file transfers between an end-user and a system.
 
-Get notified when files are available for download either by setting up a webhook or by registering your email for updates.
+**What type of encryption does CFT support?**
 
-**Audit logs** 
+CFT supports industry and IM8 compliant standards for: 
 
-View the audit logs for all your file transfer actvity.
+- Encryption at rest:......TBC
 
 
-**CFT will be available in two options:**
+- Encryption in transit:.......TBC
 
-- CFT for Backend Systems
 
-The Sender and Receiver are both systems.
+(Files in CFT are encrypted in transit and at rest with industry-standard schemes like HTTPS and Amazon Web Services (AWS) Simple Storage Service Server Side Encryption Key Management Service (S3-SSE-KMS))
```

Comment:
is this supposed to appear as a boxed item? 

my md viewer just outputs it as
"(Files in CFT are encrypted in transit and at rest with industry-standard schemes like HTTPS and Amazon Web Services (AWS) Simple Storage Service Server Side Encryption Key Management Service (S3-SSE-KMS))"